lms/templates/index.html

```diff
@@ -14,6 +14,14 @@
 
   show_partners = microsite.get_value('show_partners', True)
 
+  if self.theme_enabled() and not self.is_microsite():
+    homepage_overlay_file = 'theme-homepage_overlay.html'
+    promo_video_file = 'theme-promo_video.html'
+    extra_home_content_file = 'theme-extra_home_content.html'
+  else:
+    promo_video_file = None
```

Comment:
See comment above about initializing values with the values in the `else` block, here and throughout.